src/main/java/io/aiven/kafka/connect/common/config/FormatType.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.config;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public enum FormatType {
+
+    CSV("csv") {
+        @Override
+        public String extension() {
+            return "csv";
+        }
+    },
+//    TODO: Add a classic Json Support
```

Comment:
Could you please remove commented code ... Lets use git history instead

Reply:
Valid point 
- [ ] Remove comments